vta/python/vta/libinfo.py

```diff
@@ -15,11 +15,13 @@ def find_libvta(optional=False):
     """Find VTA library"""
     curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
     lib_search = [curr_path]
+    lib_search += [os.path.join(curr_path, "..", "..", "..", "lib",)]
```

Comment:
what was the reason for adding this path?

Reply:
In many previous versions of tvm, it copies all the generated `.so` files to the `lib` directory. Therefore, I, personally and habitually, copy the generated shared libraries there. I think I can undo the change in this line.

Reply:
I think we should remove this current line then.